site/docs/extensions/multiple-sort.md

```diff
@@ -38,6 +38,16 @@ toc: true
 
 - **Default:** `true`
 
+### multiSortStrictSearch
```

Comment:
Is `multiSortStrict` or `multiSortStrictSort` better?

Reply:
multiSortStrictSort i guess

Reply:
Yes, you both are right